src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -277,32 +283,49 @@ protected ResponseEntity retrieveAllOrganisationsByStatus(String status) {
     }
 
     protected ResponseEntity inviteUserToOrganisation(NewUserCreationRequest newUserCreationRequest, String organisationIdentifier, String userId) {
-        String orgId = removeEmptySpaces(organisationIdentifier);
 
-        Object responseBody = null;
-        validateNewUserCreationRequestForMandatoryFields(newUserCreationRequest);
-        final Organisation existingOrganisation = checkOrganisationIsActive(orgId);
-        checkUserAlreadyExist(newUserCreationRequest.getEmail());
-        List<PrdEnum> prdEnumList = prdEnumService.findAllPrdEnums();
         List<String> roles = newUserCreationRequest.getRoles();
-        validateRoles(roles);
+        ProfessionalUser professionalUser = validateInviteUserRequestAndCreateNewUserObject(newUserCreationRequest, removeEmptySpaces(organisationIdentifier), roles);
+        if (newUserCreationRequest.isResendInvite() && resendInviteEnabled) {
+            return reInviteExpiredUser(newUserCreationRequest, professionalUser, roles);
+        } else {
```

Comment:
if newUserCreationRequest.isResendInvite() is true and resendInviteEnabled = false then it should be the case here?

Reply:
in this case then it will be normal invite user flow